### PR TITLE
draft.dev - MFA SSO FusionAuth

### PR DESCRIPTION
**Pull request by bradmccarty**

### CODE REVIEW

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Combinging SSO and MFA in FusionAuth
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:2]: Title spelling  "combinging"

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+description: In this article, you'll learn how to combine single sign-on with multi-factor authentication in FusionAuth.
+author: Matt Keib
+section: Authentication
+tags: single sign on mfa multi factor authentication sso security
+icon: /img/icons/sso-mfa.svg
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:7]: I think we normally use different icons that represent single sign-on and MFA. Can we try to match to those for consistency?

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+darkIcon: /img/icons/sso-mfa-dark.svg
+---
+
+No matter how strong a password is, if an attacker wants to access what lies behind it---data, administrator access to a database or an admin panel, you name it---they eventually will.
+
+Yet there's an equally well-known fact: there's no security policy end users will not complain about, no matter how easy to implement it may be or how important the measures taken are. On top of that, the executive board always seems to side with the users instead of backing [DevSecOps](https://www.redhat.com/en/topics/devops/what-is-devsecops) and system administrators, who want to safeguard the organization and improve its security posture.
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:13]: We have 0 internal links in this article at the top. I think it's important for internal backlinks. Can you add some?

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+To integrate SSO with FusionAuth, start by registering your applications in the FusionAuth admin panel. This involves adding each application's details, such as its name and redirect URLs, to create a unique identifier for each application.
+
+When you register the application on FusionAuth, you will get a client Id and client secret that you must include in your application's code to define FusionAuth as the identity provider for that app. (There are further configurations available too, but they're beyond the scope of this article.)
+
+Once your applications are registered and configured, you can assign users to them.
+
+Navigate to the **Users** section in the admin panel, select the user in question, and add them to the desired applications. 
+
+![FusionAuth's admin panel](https://i.imgur.com/ob0hCxL.png)
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:77]: We don't host images on imgur?

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+You must assign the user to both applications for the SSO to work properly.
+
+![Assigning the user to the applications](https://i.imgur.com/2M8AmP3.png)
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:81]: We don't host images on imgur

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+Assigning users to applications in FusionAuth allows it to manage the SSO process so that users can access multiple applications with a single set of credentials.
+
+### Enabling MFA
+
+Enabling MFA is also easily achieved via an option in FusionAuth's admin panel.
+
+For this guide, tenant-level MFA is configured, which means that all apps within a specific tenant will have MFA enforced.
+
+![Enabling MFA for all the registered apps](https://i.imgur.com/2QdkJ9i.png)
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:91]: Imgur image again

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+In FusionAuth, a tenant is the highest level of grouping for users and applications. Each FusionAuth instance comes with a 'Default' tenant, which can be used to manage internal users and customize the login page. If you're offering services to multiple clients or need to separate users, applications, and groups for different purposes, you can create additional tenants to ensure proper isolation and security. 
+
+All that's left to do is for the user to register an MFA method when they log in. The screenshot below shows the screen a user would use to register an authenticator app:
+
+![MFA registration prompt](https://i.imgur.com/d3Wz2wf.png)
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:97]: Imgur image again

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+And that's all it takes to get the convenience and simplicity of SSO with the hardened access control of MFA.
+
+## Conclusion
+
+SSO gives users streamlined access to multiple applications with a single login, while MFA brings an added layer of security. This article explained how combining them gives you the best of both worlds.
```
Comment (emilyjans) [inline astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:103]: "This article explained" -- we should make present tense I think?

**Review by emilyjans — CHANGES_REQUESTED** [review]
This one is super short. Wil it impact any of our other long-form SEO content for SSO and MFA?

File: astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx (hunk 3 above)
Comment (bradmccarty) [reply astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx:13]: added some.

**Pull request closed without merging**